pkg/telemetryquerylanguage/functions/tqlotel/README.md

```diff
@@ -101,18 +101,22 @@ Examples:
 
 ## limit
 
-`limit(target, limit)`
+`limit(target, limit, priority_keys)`
 
 The `limit` function reduces the number of elements in a `pdata.Map` to be no greater than the limit.
 
 `target` is a path expression to a `pdata.Map` type field. `limit` is a non-negative integer. 
 
-The map will be mutated such that the number of items does not exceed the limit. Which items are dropped is random.
+`priority_keys` is a list of attribute keys that won't be dropped during limiting.
```

Comment:
```suggestion
`priority_keys` is a list of strings of attribute keys that won't be dropped during limiting.
```

Reply:
Should also be moved up to the same line as `target` and `limit`, doesn't need its own line.